tests/vm_backup_test.go

```diff
@@ -45,138 +65,121 @@ var _ = Describe("[smoke] VM Backup", func() {
 			r.Dump(CurrentGinkgoTestDescription().Duration)
 		}
 
+		// Deleting the backup also deletes all restores, volume snapshots etc.
+		err := DeleteBackup(timeout, backupName, r.BackupNamespace)
+		if err != nil {
```

Comment:
Maybe better expressed as `Expect(err).ToNot(HaveOccurred())` that will print the error message if an error occurred.

Reply:
Then, wen it fails it does not try to remove VM. After one or two such failures other tests may fail due too not enough resources.

Maybe I can change it to new gingko Defer(), but this is also a task for another PR.

Reply:
this is `AfterEach()`, and I think failing an `Expect()` would not run next cleanups.